Interop/ONNX/src/main/java/org/tribuo/interop/onnx/LabelOneVOneTransformer.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tribuo.interop.onnx;
+
+import ai.onnxruntime.OnnxJavaType;
+import ai.onnxruntime.OnnxTensor;
+import ai.onnxruntime.OnnxValue;
+import ai.onnxruntime.OrtException;
+import com.oracle.labs.mlrg.olcut.config.PropertyException;
+import org.tribuo.ImmutableOutputInfo;
+import org.tribuo.classification.Label;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+public final class LabelOneVOneTransformer extends LabelTransformer {
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = Logger.getLogger(LabelTransformer.class.getName());
+
+    /**
+     * Constructs a Label transformer that operates on a one v one output and produces scores via voting.
+     */
+    public LabelOneVOneTransformer() {
+        super(false);
+    }
+
+    @Override
+    public void postConfig() {
+        if (generatesProbabilities) {
+            throw new PropertyException("", "generatesProbabilities", "generatesProbabilities must not be set to true for this class.");
+        }
+    }
+
+    /**
+     * Rationalises the output of an onnx model into a standard format suitable for
```

Comment:
This should probably mention that it accept/expects similar input types to `LabelTransformer`